src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="../src/images/imac.jpeg"
+        alt="apple"
+      />
+      <div class="card__product-description">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+      <div class="card__product-code"><p>Product code: 195434</p></div>
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews"><p>Reviews: 5</p></div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-text"><p>Price:</p></div>
+        <div class="card__price-value"><p>$2,199</p></div>
+      </div>
+      <a
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="card__price">
        <div class="card__price-text"><p>Price:</p></div>
        <div class="card__price-value"><p>$2,199</p></div>
      </div>
      
      <a
```